miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -105,7 +106,6 @@ internal open class MiniAppWebView(
 
     fun destroyView() {
         stopLoading()
-        webViewClient = null
```

Comment:
@minh-rakuten I updated the `config` submodule which also updated the `targetSdkVersion` and `compileSdkVersion` to 30. It looks like in API level 30, the `webViewClient` is non-nullable.

So my fix here was just to remove setting it as `null`. Do you think this is okay, or is there another solution?

Reply:
I think the reason this was added is because our custom WebViewClient stores the Context, right? So maybe I could just reset this to the default?
```
webViewClient = WebViewClient()
```

What do you think?

Reply:
I have tested. It's absolutely fine.

Reply:
Both are okay to me because we don't know exactly what `mProvider` is executing from [here](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/webkit/WebView.java).

Google set it as non-null then I believe they already isolate the `webViewClient` reference.